GOVERNANCE.md

```diff
@@ -0,0 +1,104 @@
+
+## Roles
+
+### User
+
+Anyone who has downloaded, deployed, or operated the Aerie system to meet a specific objective. Aerie was primarily designed for space mission activity planning, modeling, and simulation, but let us know if you've found other uses for it.  
+
+### Contributor
+
+Contributors include anyone that provides input to the project. This includes code, documentation, graphics, designs, or anything else that tangibly improves the project. We encourage you to start contributing right away by joining our [Discussions](https://github.com/NASA-AMMOS/aerie/discussions) or submitting an [Issue](https://github.com/NASA-AMMOS/aerie/issues). 
+ 
+### Collaborator
+
+Subset of contributors who have been given write access to one or more of the Aerie repositories. Both contributors and collaborators can propose changes to the project via pull requests, but only collaborators can formally review and approve (merge) these requests. Any contributor who has made a non-trivial contribution should be on-boarded as a collaborator in a timely manner. 
+
+If you are planning on making a substantial contribution to the project or feel as though you should be given write access to a repository, please send a request to aerie_support@jpl.nasa.gov. 
+
+## Technical Steering Committee
+
+A subset of the collaborators forms the Technical Steering Committee (TSC). The TSC has authority over the following aspects of this project:
+
+- Technical direction and guidance
+- Project governance and process 
+- Contribution policy
+- Conduct guidelines
+- Maintaining the list of collaborators
+
+### TSC Committee Members
+- Chris Camargo ([camargo](https://github.com/camargo)), Jet Propulsion Laboratory
+- Matt Dailis ([mattdailis](https://github.com/mattdailis)), Jet Propulsion Laboratory
+- Jonathan Castello ([Twisol](https://github.com/Twisol)), University of California, Santa Cruz
+- Basak Alper Ramaswamy, Jet Propulsion Laboratory
+
+<details>
+
+<summary>Emeriti</summary>
+
+### TSC Emeriti
+- Pat Kenneally ([patkenneally](https://github.com/patkenneally)), Laboratory for Atmospheric and Space Physics
+
+</details>
+ 
+### Scope
+
+The TSC is primarily responsible for the Aerie core and UI projects:
+
+- https://github.com/NASA-AMMOS/aerie
+- https://github.com/NASA-AMMOS/aerie-ui
+
+However, the TSC also has responsibility over some projects, which are dependencies of the core projects:
+
+- https://github.com/NASA-AMMOS/aerie-docs
+- https://github.com/NASA-AMMOS/aerie-gateway
+- https://github.com/NASA-AMMOS/aerie-ts-user-code-runner
+- https://github.com/NASA-AMMOS/aerie-ampcs
+
+Addtionally, the TSC has authority over a few other "Aerie Extended Universe" projects:
+
+- https://github.com/NASA-AMMOS/aerie-cli

```

Comment:
JFYI, aerie-cli is still private

Reply:
I am waiting for the thumbs up from Parker before we go live with this one.